packages/utils/src/__mocks__/joiner/mock_data.ts

```diff
@@ -0,0 +1,295 @@
+import { JoinerServiceConfig } from "@medusajs/types"
+
+export const serviceConfigs: JoinerServiceConfig[] = [
```

Comment:
suggestion(non blocking): I would suggest to put the data in __fixture__ and then the mock here. your choice :)